src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -310,7 +310,7 @@ export class GwtCallback extends EventEmitter {
     });
 
     ipcMain.on('desktop_set_global_mouse_selection', (event, selection: string) => {
-      clipboard.writeText(selection, 'clipboard');
+      clipboard.writeText(selection, 'selection');
```

Comment:
I guess this is fine on Windows/Mac? The Electron docs doesn't really explain what these types are and that `selection` is only available on Linux but default is `clipboard`.